### DISCUSSION TITLE
Flagged account- never used it before

### DISCUSSION THREAD

**Discussion by luckylucky79** [community/community#72908] [New to GitHub]
Hi,

Ive just started a course that uses codespaces so I tried to sign up but i already have my email registered. 
Ive never used github as far as i can remember, im a total newbie with no coding experience at all. It says my account is flagged. 

Ive contacted support but it says 7 days wait :(

Any ideas what the issue could be?

**Comment by bilalmirza74** [comment]
My account has been flagged, but I don't know why. 
I raised a ticket two weeks ago but haven't received any response from GitHub :[
